Documents/API.md

```diff
@@ -132,30 +158,50 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     * `POST` - Upload HMDA data to submission
 
 
-* `/institutions/<institution>/summary`
-    * `GET` - Summary for Financial Institution, including filing information
+* `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits`
+    * `GET`  - List of all edits for a given submission, grouped by edit type
 
     Example response, with HTTP code 200:
 
     ```json
     {
-      "id": "12345",
-      "name": "First Bank",
-      "filings": [
-        {
-          "period": "2017",
-          "institutionId": "12345",
-          "status": "not-started"
-        },
-        {
-          "period": "2016",
-          "institutionId": "12345",
-          "status": "completed"
-        }
-      ]
+      "syntactical": {
```

Comment:
This is slightly different from the format I see returned when I run the endpoint. Here's the response from a file that has syntactical, validity, and quality errors:
```
{
  "syntactical": {
    "edits": [
      {
        "edit": "S020",
        "lars": [
          {
            "loanId": "8299422146"
          }
        ]
      },
      {
        "edit": "S010",
        "lars": [
          {
            "loanId": "8299422144"
          }
        ]
      }
    ]
  },
  "validity": {
    "edits": [
      {
        "edit": "V220",
        "lars": [
          {
            "loanId": "8299422145"
          }
        ]
      }
    ]
  },
  "quality": {
    "edits": [
      {
        "edit": "Q032",
        "lars": [
          {
            "loanId": "8299422166"
          },
          {
            "loanId": "8299422529"
          }
        ]
      },
      {
        "edit": "Q022",
        "lars": [
          {
            "loanId": "8299422475"
          }
        ]
      }
    ]
  },
  "macro": {
    "edits": []
  }
}
```